Sources/TerminalAPIKit/Models/Operation.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Operation.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public final class Operation: Codable {
+
+    /// the type of request after which the waiting screen shows: InputRequest, DisplayRequest, and Payment
+    public let type:OperationType
+    
+    /// For now, the Operation.Variant can only have WaitingScreen as value. This shows the waiting screen animation on the terminal after every request until you end the session.
+    public let variant:String
+    
+    /// UserInterfaceData.WaitingScreenTimeoutMs: how long the waiting screen is shown on the terminal display, in milliseconds.
+    public let userInterfaceData:UserInterfaceData
+    
+    public init(type: OperationType, variant: String, userInterfaceData: UserInterfaceData) {
```

Comment:
a docc-style comment for the initializer would be nice here (applies to multiple places in the PR)